packages/http-client-csharp/emitter/src/type/input-examples.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { InputParameter } from "./input-parameter.js";
+import {
+  InputArrayType,
+  InputDictionaryType,
+  InputLiteralType,
+  InputModelType,
+  InputNullableType,
+  InputPrimitiveType,
+  InputType,
+  InputUnionType,
+} from "./input-type.js";
+import { OperationResponse } from "./operation-response.js";
+
+interface InputExampleBase {
+  kind: string;
+  name: string;
+  description: string;
+  filePath: string;
+  rawExample: any;
+}
+
+export interface InputHttpOperationExample extends InputExampleBase {
+  kind: "http";
+  parameters: InputParameterExampleValue[];
+  responses: Map<number, OperationResponseExample>;
+}
+
+export interface InputParameterExampleValue {
+  parameter: InputParameter;
+  value: InputTypeExampleValue;
+}
+
+export interface OperationResponseExample {
+  response: OperationResponse;
+  // headers: SdkHttpResponseHeaderExample[];
```

Comment:
why is this commented out?

Reply:
I will add a TODO - this is something we currently do not support, therefore I put it as a future item